piemap.yaml

```diff
@@ -0,0 +1,40 @@
+packages:
+  - sp
+  - maps
+  - maptools
+  - rgeos 
+  - stringr
+  - dataRetrieval
+  - lubridate
+  - dplyr
+  - mapdata
+  - jsonlite
+  - geojsonio
+
+sources:
+  - scripts/piemap/map_utils.R
+  - scripts/piemap/data_utils.R
+  - scripts/piemap/draw_utils.R
+  
+targets:
+  all: 
+    depends: 
+      - figures/national_pies.png
+  
+  plot_metadata:
+    command: c(I(7), I(4.3), I(500))
+  
+  county_dots:
+    command: get_dots('cache/county_centroids_wu.json')
+    
+  us_states:
+    command: state_sp()
+  
+  us_counties:
+    command: reproject_census('data/cb_2017_us_county_5m.shp')
```

Comment:
I have no idea. Used what @jread-usgs added to the viz-scratch folder.

Reply:
From here: https://www.census.gov/geo/maps-data/data/cbf/cbf_counties.html

I paused because I was either going to add a fetch step or work more on the geojson from the viz.